src/config/note.config.ts

```diff
@@ -25,26 +37,107 @@ const schema = Joi.object({
     .integer()
     .optional()
     .label('HD_MAX_DOCUMENT_LENGTH'),
+  everyoneAccessAllowed: Joi.boolean()
+    .optional()
+    .default(true)
+    .label('HD_EVERYONE_ACCESS_ALLOWED'),
+  permissions: {
+    accessDefault: {
+      everyone: Joi.string()
+        .valid(...Object.values(DefaultAccessPermission))
+        .optional()
+        .default(DefaultAccessPermission.READ)
+        .label('HD_PERMISSION_ACCESS_DEFAULT_EVERYONE'),
+      loggedIn: Joi.string()
+        .valid(...Object.values(DefaultAccessPermission))
+        .optional()
+        .default(DefaultAccessPermission.WRITE)
+        .label('HD_PERMISSION_ACCESS_DEFAULT_LOGGED_IN'),
+    },
+    noteCreation: {
+      everyone: Joi.boolean()
+        .default(false)
+        .optional()
+        .label('HD_MAY_CREATE_NOTE_EVERYONE'),
+      loggedIn: Joi.boolean()
+        .default(true)
+        .optional()
+        .label('HD_MAY_CREATE_NOTE_LOGGED_IN'),
```

Comment:
I would prefer to use the prefix of the other permission vars. So instead of "May" I would use "PERMISSION".

Reply:
We explicitly decided against the term permission to not confuse this with note permissions.

Reply:
But it is a permission

Reply:
Anyway.. imho the names shoudl have some kind of common prefix like the database vars have.

like

```
HD_PERMISSION_DEFAULT_NOTE_ACCESS_EVERYONE
HD_PERMISSION_DEFAULT_NOTE_ACCESS_LOGGED_IN
HD_PERMISSION_CREATE_NOTE_EVERYONE
HD_PERMISSION_CREATE_NOTE_LOGGED_IN
```